migrations/20191111002157-track-index.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports = {
+  up: async function(queryInterface) {
+    await queryInterface.addIndex("Tracks", ["RecordingId"]);
+    await queryInterface.addIndex("TrackTags", ["TrackId"]);
+  },
+  down: async function(queryInterface) {
+    await queryInterface.removeIndex("Tracks", ["RecordingId"]);
+    await queryInterface.removeIndex("TrackTags", ["TrackId"]);
+  }
+};
```

Comment:
so these make a decent difference to the query plan?

Reply:
yep it uses these indexes which remove a cost  a cost of 20% due to searching via a hash prior